.github/workflows/build_and_merge.yml

```diff
@@ -1,13 +1,11 @@
-name: Test and Build Containers
+name: Build and merge containers
 
 on:
+  workflow_dispatch:
   push:
     branches:
+      - main
```

Comment:
Why add `main`?

Removing tags makes sense as we should transition to `tagged` releases for those.